### ISSUE TITLE
Dockerhub v1.4.1 error on startup

### ISSUE THREAD

**Issue by mikeseese** [textileio/lotus-devnet#40]
I'm receiving the following error when running the latest version `v1.4.1` of the dockerhub image:

```
panic: open mainnet.pi: file does not exist

goroutine 1 [running]:
github.com/GeertJohan/go%2erice.(*Box).MustString(...)
        /go/pkg/mod/github.com/!geert!johan/go.rice@v1.0.0/box.go:329
github.com/filecoin-project/lotus/build.BuiltinBootstrap(0x7f4f5b802560, 0xc000cef020, 0x0, 0xc000842400, 0x0)        
        /go/pkg/mod/github.com/filecoin-project/lotus@v1.4.1-pre1/build/bootstrap.go:21 +0x1fb
github.com/filecoin-project/lotus/node/modules.BuiltinBootstrap(0x0, 0x0, 0x0, 0x0, 0x0)
        /go/pkg/mod/github.com/filecoin-project/lotus@v1.4.1-pre1/node/modules/core.go:164 +0x25
reflect.Value.call(0x2a52a00, 0x30834a0, 0x13, 0x2dea923, 0x4, 0x5a62768, 0x0, 0x0, 0x2bcc920, 0xc0009261d0, ...)     
        /usr/local/go/src/reflect/value.go:476 +0x8c7
reflect.Value.Call(0x2a52a00, 0x30834a0, 0x13, 0x5a62768, 0x0, 0x0, 0xc000ac08f0, 0xc000926290, 0x5abaaf)
        /usr/local/go/src/reflect/value.go:337 +0xb9
github.com/filecoin-project/lotus/node.as.func2(0x5a62768, 0x0, 0x0, 0x5a62768, 0x203001, 0x2bce720)
        /go/pkg/mod/github.com/filecoin-project/lotus@v1.4.1-pre1/node/options.go:140 +0xf0
reflect.callReflect(0xc0005dfaa0, 0xc000926410, 0xc0009263f8)
        /usr/local/go/src/reflect/value.go:565 +0x32a
reflect.makeFuncStub(0x0, 0x0, 0x0, 0x0, 0x0, 0xc0005dfa10, 0xc0005dd001, 0xc000ceeff0, 0xc000926668, 0x613127, ...)
        /usr/local/go/src/reflect/asm_amd64.s:20 +0x42
reflect.Value.call(0x2a52a00, 0xc0005dfaa0, 0x13, 0x2dea923, 0x4, 0x5a62768, 0x0, 0x0, 0x10, 0x10, ...)
        /usr/local/go/src/reflect/value.go:476 +0x8c7
reflect.Value.Call(0x2a52a00, 0xc0005dfaa0, 0x13, 0x5a62768, 0x0, 0x0, 0x2bc6160, 0x2d20b01, 0xc000ce45b0)
        /usr/local/go/src/reflect/value.go:337 +0xb9
go.uber.org/dig.defaultInvoker(0x2a52a00, 0xc0005dfaa0, 0x13, 0x5a62768, 0x0, 0x0, 0xc000410910, 0x5a62768, 0x0)
        /go/pkg/mod/go.uber.org/dig@v1.10.0/dig.go:284 +0x65
go.uber.org/dig.(*node).Call(0xc00093be00, 0x3285a00, 0xc000410910, 0x3299d40, 0x2a71720)
        /go/pkg/mod/go.uber.org/dig@v1.10.0/dig.go:710 +0x27b
go.uber.org/dig.paramSingle.Build(0x0, 0x0, 0x0, 0x3299d40, 0x2a71720, 0x3285a00, 0xc000410910, 0x2a332e0, 0xc000caa250, 0xc00052e460, ...)
        /go/pkg/mod/go.uber.org/dig@v1.10.0/param.go:245 +0x2e2
go.uber.org/dig.paramObjectField.Build(...)
        /go/pkg/mod/go.uber.org/dig@v1.10.0/param.go:384
go.uber.org/dig.paramObject.Build(0x3299d40, 0x2d2f000, 0xc000416a00, 0xa, 0x10, 0x3285a00, 0xc000410910, 0x1, 0x1, 0x7f4f5b802560, ...)
        /go/pkg/mod/go.uber.org/dig@v1.10.0/param.go:311 +0x182
go.uber.org/dig.paramList.BuildList(0x3299d40, 0x2a9b420, 0xc000295130, 0x1, 0x1, 0x3285a00, 0xc000410910, 0xc000926d18, 0x5832ec, 0x10, ...)
        /go/pkg/mod/go.uber.org/dig@v1.10.0/param.go:201 +0xb8
go.uber.org/dig.(*node).Call(0xc00093bd60, 0x3285a00, 0xc000410910, 0x3299d40, 0x2da7340)
        /go/pkg/mod/go.uber.org/dig@v1.10.0/dig.go:701 +0x105
go.uber.org/dig.paramSingle.Build(0x0, 0x0, 0x0, 0x3299d40, 0x2da7340, 0x3285a00, 0xc000410910, 0x2b2c960, 0xc000112f40, 0x94, ...)
        /go/pkg/mod/go.uber.org/dig@v1.10.0/param.go:245 +0x2e2
go.uber.org/dig.paramList.BuildList(0x3299d40, 0x2b6d3e0, 0xc0001a6c60, 0x9, 0x9, 0x3285a00, 0xc000410910, 0x0, 0x0, 0x5ae1e5, ...)
        /go/pkg/mod/go.uber.org/dig@v1.10.0/param.go:201 +0xb8
go.uber.org/dig.(*Container).Invoke(0xc000410910, 0x2b6d3e0, 0x3083548, 0x0, 0x0, 0x0, 0x0, 0x0)
        /go/pkg/mod/go.uber.org/dig@v1.10.0/dig.go:432 +0x2c6
go.uber.org/fx.(*App).executeInvokes(0xc000438e70, 0x2a53f00, 0xc0002957c0)
        /go/pkg/mod/go.uber.org/fx@v1.9.0/app.go:547 +0x35f
go.uber.org/fx.New(0xc0008bf4a8, 0x3, 0x3, 0xc0005f3980)
        /go/pkg/mod/go.uber.org/fx@v1.9.0/app.go:345 +0x490
github.com/filecoin-project/lotus/node.New(0x326ebc0, 0xc0005bf580, 0xc0004106e0, 0xa, 0xa, 0xc00027e150, 0x1, 0x1)
        /go/pkg/mod/github.com/filecoin-project/lotus@v1.4.1-pre1/node/builder.go:624 +0x49c
github.com/textileio/lotus-devnet/ldevnet/node/test.mockSbBuilderOpts(0xc000503580, 0x1, 0x1, 0xc0005ba4d0, 0x1, 0x1, 0x2d10100, 0x0, 0x0, 0xc00003cd00, ...)
        /app/ldevnet/node/test/builder.go:148 +0x1492
github.com/textileio/lotus-devnet/ldevnet/node/test.MockSbBuilder(...)
        /app/ldevnet/node/test/builder.go:57
github.com/textileio/lotus-devnet/ldevnet.rpcBuilder(0xc0005ba4d0, 0x1, 0x1, 0xc0005ba401, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)     
        /app/ldevnet/ldevnet.go:169 +0xb8
github.com/textileio/lotus-devnet/ldevnet.New(0x1, 0x59682f00, 0x2f47e01, 0x0, 0x0, 0x0, 0x4, 0x0, 0x0)
        /app/ldevnet/ldevnet.go:72 +0x105
main.main()
        /app/main.go:46 +0x4f
```

**Comment by jsign** [comment]
This version isn't ready. The latest available one is 1.3.0. Lotus v1.4.1 isn't official, it's still a RC. (1.4.0 was skipped in devnet since was only a hotfix for the network outage)

**Comment by mikeseese** [comment]
Ah, thanks for the info!
